benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -149,47 +149,49 @@ def _run_vm(self):
 
     def _verify_single_vm(self, vm_name, retries=5, delay=10):
         """
-        Verifies the SSH status of a single VM with retry mechanism and logging.
+        Verifies the virtctl status of a single VM with retry mechanism
         :param vm_name: The name of the VM to verify.
         :param retries: Number of retry attempts.
         :param delay: Time to wait (in seconds) between retries.
         """
         vm_ssh = 0
+        ssh_status = None  # Initialize ssh_status to avoid potential errors
         self._virtctl.expose_vm(vm_name=vm_name)
 
         for attempt in range(retries):
             try:
-                vm_node = self._oc.get_vm_node(vm_name)
-                node_ip = self._oc.get_nodes_addresses().get(vm_node)
-
-                if node_ip:
-                    vm_node_port = self._oc.get_exposed_vm_port(vm_name)
-                    ssh_status = self._oc.get_vm_ssh_status(vm_name, node_ip, vm_node_port)
-                    vm_ssh = 1 if ssh_status == 'True' else 0
+                ssh_status = self._oc.get_vm_virtctl_status(vm_name)
+                vm_ssh = 1 if ssh_status == 'True' else 0
```

Comment:
Why do you need this separately from `ssh_status`?  You can always use `int(vm_ssh)` for the same purpose.

Reply:
In case of error we return the error message:
```
virtualmachineinstance.kubevirt.io "windows-vm-65265214-0" not found
kex_exchange_identification: Connection closed by remote host
```

Reply:
That's fine, but why does `vm_ssh` need to be distinct from `ssh_status`?

Reply:
Oh, I see...it would be cleaner to return the boolean status and the message as two return values:

```
if ok:
    return (True, '')
else:
    return (False, 'This is why it failed')

Reply:
We need 2 parameters:
1.  vm_ssh: True/False - if ssh passed successfully (Filtering in Grafana only error msg)
2.  ssh_status: error msg in case of error

Reply:
Exactly; don't overload them.

Reply:
I prefer to keep it separately for Grafana dashboard

Reply:
Can't you convert it only when you need to insert it into grafana, rather than carrying around two separate variables that could get out of sync?

Reply:
@RobertKrawitz I agree.
Use the 'virtctl_status' string variable (instead of 'ssh_status') with the value 'True' if successful, or an error message if it fails.
Additionally, add the 'virtctl_vm' integer value (instead of 'vm_ssh') only for Grafana.